tests/utils/get-featured-articles.test.js

```diff
@@ -0,0 +1,45 @@
+import { getFeaturedArticles } from '../../src/utils/setup/on-create-page';
+
+it('should correctly find featured articles given a set of requested articles', () => {
```

Comment:
Should we think about capping the list to 4 and 3 for home and learn too?

Reply:
Yes we definitely should. The home page is flexible to support <= 4, but the learn page should really be 3 (otherwise it would break the design?).